annotation-processor/src/main/java/org/hibernate/validator/ap/checks/ConstraintCheckFactory.java

```diff
@@ -189,7 +201,9 @@ public ConstraintCheckFactory(Types typeUtils, ConstraintHelper constraintHelper
 						new TargetCheck( annotationApiHelper ),
 						new ConstraintValidatorCheck( constraintHelper, annotationApiHelper ),
 						new AnnotationTypeMemberCheck( annotationApiHelper, typeUtils ),
-						new CrossParameterConstraintCheck( annotationApiHelper, constraintHelper, typeUtils )
+						new CrossParameterConstraintCheck( annotationApiHelper, constraintHelper, typeUtils ),
+						new AnnotationUserMessageCheck( annotationApiHelper ),
+						new AnnotationDefaultMessageCheck( annotationApiHelper, elementUtils )
```

Comment:
How comes the default message check is given more than once? I'd have only expected it for `@Constraint` on an annotation type.